.github/labeler.yml

```
@@ -44,9 +44,9 @@
 - 'src/cpp/include/openvino/genai/lora_adapter.hpp'
 - 'src/cpp/src/lora_adapter.cpp'
 - 'src/cpp/src/lora_helper.cpp'
-- 'src/cpp/src/lora_helper.hpp'
+- 'src/cpp/src/helper.hpp'
```

```suggestion
- 'src/cpp/src/lora/helper.hpp'
```
Other paths also need fixing here

---

[.github/labeler.yml](https://github.com/openvinotoolkit/openvino.genai/pull/2146/files/82cff0583b1897a03c8e05e83c5039e3ea2f2348#diff-a22c263686553013feaeb0677d26eeb0b8778a756c4311c1fce13384258026aa) is planned to be updated as a second step towards Smart CI